scripts/i18n.js

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Updates the translations of the project, located in the
+ * directory locales/
+ */
+
+import fs from 'node:fs/promises';
+import fsSync from 'node:fs';
```

Comment:
Let's change all `fs` usages to `node:fs/promises`.

Our `webpack.config.js` is using ESM as well. So feel free to to top-level async/await.